datalakehouse-api/FAST_API_For_BronzeRawData.py

```
@@ -0,0 +1,207 @@
+            raise HTTPException(status_code=403, detail={"message": "Only SELECT allowed"})
+    if not (q_upper.startswith("SELECT") or q_upper.startswith("WITH")):
+        raise HTTPException(status_code=400, detail={"message": "Only SELECT/WITH allowed"})
+    try:
+        for tname, path in Bronze_PATHS.items():
+            spark.read.format("hudi").load(path).createOrReplaceTempView(tname)
+        df = spark.sql(sql_query)
+        if request.limit:
+            df = df.limit(request.limit)
+        data = [row.asDict(recursive=True) for row in df.collect()]
+        return {"status": "success", "code": 200, "query": sql_query, "row_count": len(data), "data": data}
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)